diff --git a/plugins/cfpython/cfpython.cpp b/plugins/cfpython/cfpython.cpp
index 04373a008..a1eac74ea 100644
--- a/plugins/cfpython/cfpython.cpp
+++ b/plugins/cfpython/cfpython.cpp
@@ -1492,7 +1492,7 @@ static PyObject* PyInit_Crossfire(void)
 }
 
 extern "C"
-int initPlugin(const char *iversion, f_plug_api gethooksptr) {
+CF_PLUGIN int initPlugin(const char *iversion, f_plug_api gethooksptr) {
     PyObject *m;
     /* Python code to redirect stdouts/stderr. */
     const char *stdOutErr =
